ShiftLogger.Bata13/STUDY.ASP.ShiftLoggerTryThreeUserInterface/ProgramEngine.cs

```diff
@@ -0,0 +1,200 @@
+﻿using Spectre.Console;
+using STUDY.ASP.ShiftLoggerTryThree.Models;
+using System.Net.Http.Json;
+
+namespace STUDY.ASP.ShiftLoggerTryThreeUserInterface;
+internal class ProgramEngine
+{
+    public static void ViewAllShiftLogs(HttpClient client, string ApiBaseUrl)
+    {
+        var shiftLogs = client.GetFromJsonAsync<ShiftLogger[]>(ApiBaseUrl).Result;
+        if (shiftLogs != null)
```

Comment:
It should check if `shiftLogs.Count() != 0`, because you transform it into IEnumerable, which will never be null in this case, so "Shift logs not found" will never be reached 😁